bitnami/kong/3/debian-11/docker-compose.yml

```diff
@@ -1,6 +1,7 @@
 version: '2'
 services:
   postgresql:
+    # New change
```

Comment:
Is this needed really?

Reply:
It is needed to trigger the tests. To push the changes, we have to delete this line.